src/functions/helpers.ts

```diff
@@ -74,15 +74,28 @@ export function tryToNumber(
   }
 }
 
+function cellTypeToTranslatedString(cellValue: CellValue): string {
+  switch (typeof cellValue) {
+    case "number":
+      return _t("a number");
+    case "string":
+      return _t("a string");
```

Comment:
it will lead to better translations if the sentence isn't cut in half.
I'd "duplicate" the full error messages
```suggestion
function cellTypeToTranslatedString(cellValue: CellValue): string {
  switch (typeof cellValue) {
    case "number":
      return _t("Function [[FUNCTION_NAME]] expects number values for %s, but got a number.");
    case "string":
      return _t("Function [[FUNCTION_NAME]] expects number values for %s, but got a string.");
```